internal/pkg/utils/testproject/snapshot.go

```diff
@@ -182,6 +182,18 @@ func (p *Project) NewSnapshot() (*fixtures.ProjectSnapshot, error) {
 		return request.SendOrErr(ctx, p.storageAPIClient)
 	})
 
+	// Storage Files
+	var files []*storageapi.File
+	grp.Go(func() error {
+		request := storageapi.
+			ListFilesRequest().
+			WithOnSuccess(func(_ context.Context, _ client.Sender, apiFiles *[]*storageapi.File) error {
+				files = append(files, *apiFiles...)
```

Comment:
nestaci `files = *apiFiles`? ak dobre pozeram, tak je to v kode pouzite iba raz

Reply:
Jasně. 🙈